src/store/modules/orderLookup/actions.ts

```diff
@@ -0,0 +1,357 @@
+        }
+
+        if (query.json.params.start && query.json.params.start > 0) stateOrders = stateOrders.concat(orders)
+        else stateOrders = orders
+        this.dispatch("product/getProductInformation", { orders });
+        this.dispatch("util/fetchShipmentMethodTypeDesc", shipmentMethodTypeIds)
+      } else {
+        showToast(translate("Failed to fetch orders"));
+      }
+    } catch(error) {
+      logger.error(error)
+      showToast(translate("Failed to fetch orders"));
+    }
+    commit(types.ORDERLOOKUP_LIST_UPDATED, { orders: stateOrders, orderCount, itemCount });
+    return resp;
+  },
+
+  async getOrderDetails({ commit }, orderId) {
+    let order = {} as any;
+
+    try {
+      const orderHeader = await OrderLookupService.performFind({
+        inputFields: {
+          orderId
+        },
+        viewSize: 1,
+        entityName: "OrderHeader"
+      })
+
+      if(hasError(orderHeader) && orderHeader.data.count <= 0) {
+        throw "Failed to fetch order information"
+      }
+
+      const apiPayload = [{
+        inputFields: {
+          orderId,
+          roleTypeId: "BILL_TO_CUSTOMER"
+        },
+        viewSize: 1,
+        filterByDate: "Y",
+        fieldList: ["partyId", "roleTypeId"],
+        entityName: "OrderRole"
+      }, {
+        inputFields: {
+          orderId,
+          contactMechPurposeTypeId: ["BILLING_LOCATION", "BILLING_EMAIL", "PHONE_BILLING"],
+          contactMechPurposeTypeId_op: "in"
+        },
+        viewSize: 20,
+        fieldList: ["contactMechPurposeTypeId", "contactMechId"],
+        entityName: "OrderContactMech"
+      }, {
+        inputFields: {
+          orderId
+        },
+        viewSize: 20,
+        filterByDate: "Y",
+        fieldList: ["orderIdentificationTypeId", "orderId", "idValue"],
+        entityName: "OrderIdentification"
+      }, {
+        inputFields: {
+          orderId,
+          attrName: ["customerId", "municipio"],
+          attrName_op: "in",
+          attrName_ic: "Y"
+        },
+        viewSize: 10,
+        fieldList: ["attrName", "attrValue"],
+        entityName: "OrderAttribute"
+      }, {
+        inputFields: {
+          orderId
+        },
+        viewSize: 20,
+        orderBy: "changeDatetime",
+        entityName: "OrderFacilityChange"
+      }, {
+        inputFields: {
+          orderId,
+          statusId: ["ORDER_COMPLETED", "ORDER_APPROVED"],
+          statusId_op: "in"
+        },
+        viewSize: 2,
+        entityName: "OrderStatus"
+      }, {
+        inputFields: {
+          orderId,
+          statusId: "PAYMENT_CANCELLED",
+          statusId_op: "notEqual"
+        },
+        viewSize: 20,
+        fieldList: ["paymentMethodTypeId", "maxAmount", "statusId"],
+        entityName: "OrderPaymentPreference"
+      }, {
+        inputFields: {
+          orderId
+        },
+        viewSize: 20,
+        orderBy: "facilityId DESC",
+        entityName: "OrderItemShipGroupAndShipGroupAssoc"
```

Comment:
I think we can remove this call, as we are already fetching detail from OrderItemShipGroupAndFacility later part of logic.